rules/windows/defense_evasion_windows_firewall_profile_disabled.toml

```diff
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2021/10/15"
+maturity = "production"
+updated_date = "2021/10/15"
+
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when a user disables the Windows Firewall Profile to help evade defense.
+"""
+false_positives = [
+    """
+    Windows Firewall Profile being disabled may be performed by a system administrator. Verify whether the user identity, 
```

Comment:
Windows Firewall Profile being disabled may be performed by a system administrator -> Windows Firewall can be disable by a system administrator